ginga/table/AstroTable.py

```diff
@@ -23,7 +23,7 @@ class AstroTableHeader(Header):
     pass
 
 
-class AstroTable(Callback.Callbacks):
+class AstroTable(ViewerObjectBase):
```

Comment:
:+1:  I thought about doing something like this but I wasn't sure what to name the base class. I am glad you also came to the same conclusion.